server/src/main/java/com/exacaster/lighter/application/sessions/SessionService.java

```diff
@@ -31,40 +35,56 @@ public class SessionService {
     private final StatementHandler statementHandler;
 
     public SessionService(ApplicationStorage applicationStorage,
-            StatementStorage statementStorage, Backend backend,
-            StatementHandler statementHandler) {
+                          StatementStorage statementStorage, Backend backend,
+                          StatementHandler statementHandler) {
         this.applicationStorage = applicationStorage;
         this.statementStorage = statementStorage;
         this.backend = backend;
         this.statementHandler = statementHandler;
     }
 
+    //TODO do we wanna return just Session or PermSession as well?
     public List<Application> fetch(Integer from, Integer size) {
-        return applicationStorage.findApplications(ApplicationType.SESSION, from, size);
+        return applicationStorage.findApplications(EnumSet.of(ApplicationType.SESSION, ApplicationType.PERMANENT_SESSION), from, size);
```

Comment:
`EnumSet.of(ApplicationType.SESSION, ApplicationType.PERMANENT_SESSION)` could be move to a private constant - in that case we would not initiate a new EnumSet for each `fetch` invocation.

That constant could also be reused on `deleteOne`

Reply:
done